Alcatraz/Categories/NSImage+Alcatraz.m

```diff
@@ -0,0 +1,40 @@
+// NSImage+Alcatraz.m
+//
+// Copyright (c) 2016 Roman Gardukevich
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+#import "NSImage+Alcatraz.h"
+
+@implementation NSImage (Alcatraz)
+-(NSInteger)frameCount{
+    // Find the NSBitmapImageRep representation to check frame count GIF-specific value
+    // returns 1 for common image
+    
+    for (NSImageRep * rep in self.representations){
```

Comment:
Please remove the whitespace after `*`, and add one before `{` to be consistent with the rest of the codebase.

```objectivec
for (NSImageRep *rep in self.representations) {
```